pkg/virt-controller/services/template_test.go

```diff
@@ -3448,6 +3448,83 @@ var _ = Describe("Template", func() {
 
 		})
 
+		Context("virtiofs container qos", func() {
+
+			var vmi *v1.VirtualMachineInstance
+			BeforeEach(func() {
+				vmi = &v1.VirtualMachineInstance{
+					Spec: v1.VirtualMachineInstanceSpec{
+						Volumes: []v1.Volume{
+							{
+								Name: "fakeVol1",
+							},
+						},
+						Domain: v1.DomainSpec{
+							CPU: &v1.CPU{
+								Cores:   1,
+								Sockets: 1,
+								Threads: 1,
+							},
+							Devices: v1.Devices{
+								DisableHotplug: true,
+								Filesystems: []v1.Filesystem{
+									{
+										Name: "fakeVol1",
+									},
+								},
+							},
+						},
+					},
+				}
+			})
+
+			DescribeTable("should container in QOSGuaranteed group ", func(req, lim kubev1.ResourceList, dedicatedCpu, quaranteedQos bool) {
+				kvConfig := &v1.KubeVirtConfiguration{
+					SupportContainerResources: []v1.SupportContainerResources{
+						{
+							Type: v1.VirtioFS,
+							Resources: kubev1.ResourceRequirements{
+								Requests: kubev1.ResourceList{},
+								Limits:   kubev1.ResourceList{},
+							},
+						},
+					},
+				}
+				kvConfig.SupportContainerResources[0].Resources.Requests = req
+				kvConfig.SupportContainerResources[0].Resources.Limits = lim
+				clusterConfig, _, _ := testutils.NewFakeClusterConfigUsingKVConfig(kvConfig)
+				qosGroupName := kubev1.PodQOSGuaranteed
+				if !quaranteedQos {
+					qosGroupName = kubev1.PodQOSBurstable
+				}
+				vmi.Spec.Domain.CPU.DedicatedCPUPlacement = dedicatedCpu
+				vmi.Status.QOSClass = &qosGroupName
```

Comment:
Please drop this line and then you should be able to drop `quaranteedQos` all together